source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -23,17 +24,48 @@ OpenGLSurface: abstract class extends GpuCanvas {
 	_unbind: abstract func
 	drawLines: override func ~explicit (pointList: VectorList<FloatPoint2D>, pen: Pen) {
 		this _bind()
-		this context backend setViewport(this viewport)
+		this context backend setViewport(IntBox2D new(this size))
 		this context backend enableBlend(false)
-		this context drawLines(pointList, this _projection * this _toLocal, pen)
+		this context drawLines(pointList, this _createProjection(this size toFloatVector2D(), 0.0f) * this _toLocal, pen)
```

Comment:
Creating an isometric projection matrix for 2D drawing in reference coordinates.